src/getting_started.md

```diff
@@ -287,29 +289,13 @@ Now that the board is connected and you have verified that the kernel compiles
     $ cd boards/<your board>
     $ make
 
-Generally boards are programmed with either `make program` or `make flash`. Try
-`make program` first:
-
-    $ make program
-
-If that is not available, you can use `make flash`:
+Boards provide the target `make install` as the recommended way to load the
+kernel.
 
-    $ make flash
+    $ make install
 
 You can also look at the board's README for more details.
 
-> #### Why both `make program` and `make flash`?
->
-> While these commands do the same thing, the way they go about it is very
-> different.
->
-> The `make program` version communicates with the board via a serial connection
-> and a bootloader running on the board. You may need to manually enter the
-> bootloader when using `make program`.
->
-> The `make flash` version uses a JTAG debugger to communicate with the chip and
-> flash the kernel binary directly to the chip.
-
```

Comment:
I think this is still useful information to include. Especially as a little 'quote block aside'. Can maybe add a sentence at the end now that says

> The `make install` target will automatically choose between `flash` or `program` based on which is the more typical use case for the given board.

Reply:
Do new users really care? If this wasn't already there, would getting started be the place you would add it? I decided it wouldn't be.